### PR TITLE
Reduce images 04

### PR DESCRIPTION
**Pull request by agamm**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: https://github.com/demisto/content/pull/17059

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ImpossibleTraveler/ReleaseNotes/1_2_5.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Playbooks
+##### Impossible Traveler
+- Updated the Docker image to: *demisto/py3-tools:0.0.1.25751*.
```
Comment (yuvalbenshalom) [inline Packs/ImpossibleTraveler/ReleaseNotes/1_2_5.md:4]: the release note should state the script/integration that was updated, as the playbook itself was not updated.
see ` FetchIndicatorsFromFile/ReleaseNotes/1_0_5.md` in your PR as a good example.

File: Packs/Intezer/ReleaseNotes/1_2_6.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### IntezerScanHost
+- Updated the Docker image to: *demisto/py3-tools:0.0.1.25751*.
```
Comment (yuvalbenshalom) [inline Packs/Intezer/ReleaseNotes/1_2_6.md:4]:
```suggestion
- Updated the Docker image to: *demisto/py3-tools:0.0.1.25751*.
#### Integrations
##### IntezerV2
- Updated the Docker image to: *demisto/py3-tools:0.0.1.25751*.
```

File: Packs/JARM/Integrations/JARM/JARM.yml [hunk 3]
```diff
@@ -65,7 +65,7 @@ script:
   script: '-'
   type: python
   subtype: python3
-  dockerimage: demisto/pyjarm:1.0.0.17085
+  dockerimage: demisto/py3-tools:0.0.1.25751
```
Comment (yuvalbenshalom) [inline Packs/JARM/Integrations/JARM/JARM.yml:68]: missing release notes

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
see my comments

File: Packs/JARM/Integrations/JARM/JARM.yml (hunk 3 above)
Comment (agamm) [reply Packs/JARM/Integrations/JARM/JARM.yml:68]: Added, there were none before.

**Comment by agamm** [comment]
@yuvalbenshalom done.

File: Packs/Intezer/ReleaseNotes/1_2_6.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### IntezerV2
```
Comment (yuvalbenshalom) [inline Packs/Intezer/ReleaseNotes/1_2_6.md:2]:
```suggestion
##### Intezer v2
```
my bad

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]

**Comment by yuvalbenshalom** [comment]
please take a look at the unit tests and the test playbooks failures.

**Review by yuvalbenshalom — APPROVED** [review]

**Pull request merged**